src/Dapr.Actors/Runtime/DefaultActorTimerManager.cs

```diff
@@ -77,5 +91,24 @@ private async ValueTask<string> SerializeReminderAsync(ActorReminder reminder)
                 reminder.Ttl);
             return await info.SerializeAsync();
         }
+
+        private async ValueTask<ActorReminder> DeserializeReminderAsync(Stream stream, ActorReminderToken token)
+        {
+            if (stream == null)
+            {
+                throw new ArgumentNullException(nameof(stream));
+            }
+            var streamContents = new byte[stream.Length];
+            stream.Read(streamContents);
+            if(Encoding.ASCII.GetString(streamContents) == "null")
+            {
+                return null;
+            }
+            stream.Seek(0, SeekOrigin.Begin);
```

Comment:
There are no guarantees that the `Stream` provided supports seeking; if we really need to do this, the capability should be verified and, if not, a copy of the stream (e.g. a `MemoryStream`) should be made.  If we were to know the JSON deserializer handles `null`, then it seems we would have no need for this.

Reply:
Removed the Seeking code, and handled the case in JsonSerializer itself.